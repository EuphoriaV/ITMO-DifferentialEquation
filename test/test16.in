1.49
((((x - 3.5) * (x - x)) * ((x + 9.57) - (x + 7.1))) - (x * ((10.7 + x) + (x - x))))
2.61
3.27