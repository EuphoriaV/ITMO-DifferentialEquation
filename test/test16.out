-0.5694213